content/webapp/pages/whats-on.tsx

```diff
@@ -21,7 +23,7 @@ import {
 } from '@weco/common/services/prismic/opening-times';
 import {
   cafePromo,
-  shopPromo,
+  // shopPromo,
```

Comment:
I have no idea why TypeScript is suddenly angry about this – it's used in some commented-out code further down, which is why I haven't removed it entirely.